lib/cli/config-schema.js

```diff
@@ -4,7 +4,8 @@ const Joi = require('@hapi/joi');
 const JoiConfigSchema = Joi.object({
   order: Joi.array()
     .min(0)
-    .unique(),
+    .unique()
+    .required(),
```

Comment:
Will this fail on partial configs?

Reply:
@camacho This will fail the validation, `order` is the only required property at the moment, perhaps make it optional?

Reply:
I think optional is best for this case. But we should definitely validate it is an array.

Reply:
Fixed. Made it optional and if passed, validate it as an array; a bit more forgiving.